docs/source/scilla-in-depth.rst

```diff
@@ -1081,14 +1081,20 @@ below, ``Any`` can be of type ``IntX``, ``UintX``, ``String``, ``ByStr20`` or
   The return type is ``Option String``. On success, ``Some addr`` is returned and on invalid inputs ``None`` is returned.
 
 - ``builtin alt_bn128_G1_add p1 p2``. The builtin takes two points ``p1``, ``p2`` on the ``alt_bn128`` curve and returns 
-  the sum of the points in the underlying group G1. The input points and the result point are each a ``Pair {Bystr32 ByStr32}``.
+  the sum of the points in the underlying group G1. The input points and the result option of a point are each a ``Some (Pair Bystr32 ByStr32)``.
```

Comment:
Let's tweak this description a bit to match the style of the other option-valued builtins. For instance, just above this paragraph it says:

> The return type is ``Option String``. On success, ``Some addr`` is returned and on invalid inputs ``None`` is returned.